README.md

```diff
@@ -8,54 +8,81 @@
 The ainft-js is typescript SDK to interact with AIN blockchain and create and manage AINFT.
 
 ## AINFT Factory
-The AINFT Factory is a component consisting of AINFT Factory server and ainft-js.  AINFT Factory supports the following two features:
+
+The AINFT Factory is a component consisting of AINFT Factory server and ainft-js. AINFT Factory supports the following two features:
+
 - AINFT: Supports creating and managing AINFT, the NFT of the Ain blockchain.
 - Tokenomics: Supports functions for activating tokenomics in NFT communities.
 
 You can see reference about AINFT Factory: https://docs.ainetwork.ai/ainfts/ainft.
 
-## Getting start
+## Installation
 
-```bash
+```sh
 npm install @ainft-team/ainft-js
```

Comment:
could we use 'yarn' as a default as well?

Reply:
done!